### ISSUE TITLE
Add some more built-in colour schemes

### ISSUE THREAD

**Issue by christianp** [coursebuilder-ncl/makecourse#229]
Since rewriting the default theme, there are now only "black text on white" and "white text on black" built-in colour schemes.

The old theme had a pastel theme, which we should make available. Talking to students might bring up some more options that would be useful.

This is mentioned in #146.
